src/renderer/components/Validator/Validator.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import {ButtonPrimary, ButtonDestructive} from "../Button/ButtonStandard";
+import {AddButton} from "../Button/ButtonAction";
+import {ValidatorCard} from "../Cards/ValidatorCard";
+import {NodeCard} from "../Cards/NodeCard";
+import {useState} from "react";
+import BN from "bn.js";
+
+export interface IStatsType{
+    roi: number;
+    balance: BN | number;
+    uptime: number;
+}
+export interface IBeaconNode{
+    id: string;
+    url: string;
+    respTime: number;
+}
+export interface IValidatorProps {
+    name: string;
+    stats: IStatsType;
+    beaconNodes: IBeaconNode[];
+    onBeaconNodeClick: () => void;
+    onRemoveClick: () => void;
+    onDetailsClick: () => void;
+    onAddNodeClick: () => void;
+}
+
+export const Validator: React.FunctionComponent<IValidatorProps> = (
+    props: IValidatorProps) => {
+    const [nodeStatus, setNodeStatus] = useState(false);
+
+    let numOfNodes = 0;
+    for (let i = 0; i < props.beaconNodes.length; i++) {
+        if (props.beaconNodes[i].id) numOfNodes++;
+    }
+
+    const getShowMore = (): void=>{
+        !nodeStatus ? setNodeStatus(true) : setNodeStatus(false);
+    };
+
+    return(
+        <div className="validator-container">
+            <div className="validator-stats">
+                <h2>{props.name}</h2>
+                <div className="validator-stats-container" >
+                    <ValidatorCard value={props.stats.roi} title="Return (ETH)" type="ROI"/>
+                    <ValidatorCard value={props.stats.balance} title="Balance" type="ETH"/>
+                    <ValidatorCard value={props.stats.uptime} title="Uptime" type="DAYS"/>
```

Comment:
type is Uptime

Reply:
number should be formatted to 91 days or 1year etc, but we can deal with that later